index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for express-openid-connect
+
+import { AuthorizationParameters, TokenSet } from 'openid-client';
+import { Request, RequestHandler } from 'express';
+
+interface AuthParams {
+    issuerBaseURL?: string;
+    baseURL?: string;
+    clientID?: string;
+    clientSecret?: string;
+    clockTolerance?: number;
+    getUser?: (tokenSet: TokenSet) => any;
```

Comment:
Should this really be `any`? This function should return `undefined` or an `object` representing the user. Can we be more explicit here?

Reply:
Does it have to be an object or can it be a string or something else?

Reply:
Here's the default `getUser` implementation:

https://github.com/auth0/express-openid-connect/blob/master/lib/getUser.js

We should definitely be strict on what it gets back. Following this up to the parent library, by default, that returns `IdTokenClaims`, which extends `UserinfoResponse`. I would probably use the latter for now.